app/models/manageiq/providers/tag_manager.rb

```diff
@@ -0,0 +1,6 @@
+module ManageIQ::Providers
+  class TagManager < BaseManager
+    include SupportsFeatureMixin
+    has_many :resources, :foreign_key => :ems_uid, :dependent => :destroy
```

Comment:
Would rather not introduce *another* unique identifier field, but I'm not sure what the ems_uid here represents.